src/windows/find_tools.rs

```diff
@@ -199,6 +203,47 @@ mod impl_ {
         include: Vec<PathBuf>,
     }
 
+    struct LibraryHandle(HMODULE);
+
+    impl LibraryHandle {
+        fn new(name: &[u8]) -> Option<Self> {
+            let handle = unsafe { LoadLibraryA(name.as_ptr() as _) };
+            (!handle.is_null()).then(|| Self(handle))
+        }
+    }
+
+    impl Drop for LibraryHandle {
+        fn drop(&mut self) {
+            unsafe { FreeLibrary(self.0) };
+        }
+    }
+
+    fn is_amd64_emulation_supported() -> bool {
+        let kernel32 = LibraryHandle::new(b"kernel32.dll\0").unwrap();
```

Comment:
Let's return `false` on failure to load `kernel32`?

Or use some fallback, or return error, panicking not a good way to report error.